### DISCUSSION TITLE
v7.2.2 Release of PowerShell

### DISCUSSION THREAD

**Discussion by daxian-dbw** [PowerShell/PowerShell#17010] [Show and tell]
### Build and Packaging Improvements

<details>

<summary>

<p>Bump .NET SDK to 6.0.201</p>

</summary>

<ul>
<li>Update WiX file (Internal 19460)</li>
<li>Update .NET SDK version to 6.0.201 (Internal 19457)</li>
<li>Update experimental feature JSON files (#16838)</li>
<li>Ensure Alpine and ARM SKUs have <code>powershell.config.json</code> file with experimental features enabled (#16823)</li>
<li>Update the <code>vmImage</code> and PowerShell root directory for macOS builds (#16611)</li>
<li>Update macOS build image and root folder for build (#16609)</li>
<li>Remove WiX install (#16834)</li>
<li>Opt-in to build security monitoring (#16911)</li>
<li>Add SBOM manifest for release packages (#16641, #16711)</li>
<li>Add Linux package dependencies for packaging (#16807)</li>
<li>Switch to our custom images for build and release (#16801, #16580)</li>
<li>Remove all references to <code>cmake</code> for the builds in this repo (#16578)</li>
<li>Register NuGet source when generating <code>CGManifest</code> (#16570)</li>
</ul>

</details>

[7.2.2]: https://github.com/PowerShell/PowerShell/compare/v7.2.1...v7.2.2

### SHA256 Hashes of the release artifacts

- hashes.sha256
  - 2B60FBF1AEF3D8487CD0E9768AF7AEAD26DC4EC6B60477EDDFE6CFE29D962243
- powershell_7.2.2-1.deb_amd64.deb
  - 800C8676B1B346DF51B68E564E5C3B08A5F1B4ADC66DCF304004CE8E7D747C72
- powershell-7.2.2-1.rh.x86_64.rpm
  - F3B54FE05DB54E1E19C7513F0C883FA6490D925D28169B06F774394E8F751430
- powershell-7.2.2-linux-alpine-x64.tar.gz
  - F3ABFAF4013155CD09511763932043D9449B36D9929C2DD23DAE056942A21FA0
- powershell-7.2.2-linux-arm32.tar.gz
  - 5CCB8A4DEAB99A432F5AA31C77164B6282D01E38F51D25DB7B83218DF669634B
- powershell-7.2.2-linux-arm64.tar.gz
  - FBF74576C8B6BD92E9AAB35A8794C8BF254D1B3D0B7DE476CCE5299139F37114
- powershell-7.2.2-linux-x64.tar.gz
  - 3F6B3C1A9391E0C23E06E534DA321E12AE883A64D99CB1DD738B274D97075891
- powershell-7.2.2-linux-x64-fxdependent.tar.gz
  - 21B416645BC4B2BDF83B0A145FD48AD71EBD192ACD6271DA1860E926A1724D89
- powershell-7.2.2-osx-arm64.pkg
  - 2174808F72FF5457F13530A7121608C050DC2203B1A21FEC057F27A10CE5FBD7
- powershell-7.2.2-osx-arm64.tar.gz
  - D55CCF182273B23FA9FA0DADCE51C729757F2941077BC020B445285A92EB4D7D
- powershell-7.2.2-osx-x64.pkg
  - 289093BE09B6BBFBF0F41F59F1881BDC97620D979D8EF846A9E110CAA5F07A7A
- powershell-7.2.2-osx-x64.tar.gz
  - EEE758200657E95DA773A0541ADA537EF47503B6F54794E68B2E75D7AE94DE85
- PowerShell-7.2.2-win-arm32.zip
  - AD7B7830467260320B73BE6BE0F22768777E13B1945DB4DBDA67AEB503E51B21
- PowerShell-7.2.2-win-arm64.zip
  - 1BC41DBCA0E180C7E267DF655AD1F0C3FFC6B8858F43588CD04CE101DA706568
- PowerShell-7.2.2-win-fxdependent.zip
  - BED77C768AC4F1C86084984B3D741DC9F04831281646E106E9AC91319744A872
- PowerShell-7.2.2-win-fxdependentWinDesktop.zip
  - 8DFE0B8CF3B8AF11F29597EB7605AD5A8252DD9DBB8BF7DF88CCA0E67162861F
- PowerShell-7.2.2-win-x64.msi
  - 447B49FB3DEAF1D1B632711A48978F18AE43F0A2FE4CDBA007218026A6E39010
- PowerShell-7.2.2-win-x64.zip
  - E639B5F47D6D39B5942DF70ADA4A69A1C85616747622582EEDFB197B8CCAD298
- PowerShell-7.2.2-win-x86.msi
  - C97BCE05770D85EBA7CF5EC2E129CDF4C10C1AD2457891E350302A9F24508F7A
- PowerShell-7.2.2-win-x86.zip
  - 4964B5EAD228B518197E528FDA9E5D771037896FA8091AB4C2BD71B2E0A3A021
- powershell-lts_7.2.2-1.deb_amd64.deb
  - BF822A5033403C396126818985FEE79AF163B62BB015603D81400E90AC9E32C7
- powershell-lts-7.2.2-1.rh.x86_64.rpm
  - 0A6FE25529BF21EA4903A97F3C088E0D0C462868EB27BAA4F42AAE89D0A448AE
- powershell-lts-7.2.2-osx-arm64.pkg
  - 7D1D83226B2D687F6D9A2EDF0EA24A0CB60206368D386F0249864ACC62698E98
- powershell-lts-7.2.2-osx-x64.pkg
  - 1C4116208455A9BDAF2FBAE067F07F4A10BFA88ECBB4104CC52A4166FDF318A5

<hr /><em>This discussion was created from the release <a href='https://github.com/PowerShell/PowerShell/releases/tag/v7.2.2'>v7.2.2 Release of PowerShell</a>.</em>

**Comment by kclemens1978** [comment]
Do I download all the power shells separate?

**Reply by Zerotask** [reply]
You just download the one for your OS & architecture.

**Reply by Myloveken** [reply]
Good👏

**Comment by tami6713** [comment]
エラー	1	ファイル CustomerForm.resx を処理できませんでした。インターネットまたは制限付きゾーン内にあるか、ファイルに Web のマークが付いているためです。これらのファイルを処理するには、Web のマークを削除してください。	Order
